dev/.docker/goaws.yaml

```diff
@@ -0,0 +1,10 @@
+Dev:                                # Another environment
```

Comment:
Get rid of `# Another environment` comment

Reply:
Fixed in last commit